basket/news/tasks.py

```diff
@@ -197,8 +197,11 @@ def wrapped(self, *args, **kwargs):
     return wrapped
 
 
-def gmttime():
-    d = datetime.datetime.now() + datetime.timedelta(minutes=10)
+def gmttime(basetime=None):
```

Comment:
Is it reasonable to extend this function to pass in a basetime?

Reply:
Yeah. Looks great to me. DRY :)